src/SDKs/CognitiveServices/dataPlane/Vision/FormRecognizer/FormRecognizer/generate.ps1

```diff
@@ -0,0 +1 @@
+Start-AutoRestCodeGeneration -ResourceProvider "cognitiveservices/data-plane/FormRecognizer" -AutoRestVersion "latest"
```

Comment:
Was this code generated using the generate.ps1 script? If yes, there should be a .txt and SdkInfo_*.cs file generated, please commit these files too

Reply:
The code was not generated using generate.ps1. Once I am able to do that I will add those files you mention. The code is generated using AutoRest command separately.

Reply:
If the spec is merged, please regenerate the code using this script and commit all the changes generated. We expect a .txt and an SdkInfo_*.cs file to be generated.

Reply:
done.